tests/test_manager.py

```diff
@@ -63,30 +63,31 @@ def test_init_app(self):
         :class:`flask.ext.restless.APIManager` object.
 
         """
-        manager = APIManager()
-        manager.init_app(self.flaskapp, session=self.session)
-        manager.create_api(self.Person, app=self.flaskapp)
+        manager = APIManager(session=self.session)
+        manager.create_api(self.Person)
+        manager.init_app(self.flaskapp)
```

Comment:
You have changed the order of function calls in this test, making it identical to the test `test_creation_api_without_app_dependency` below. I would like to maintain the functionality that you have removed here: initializing the app first, creating the API second.